packages/create-audius-app/tests/e2e/react.test.ts

```diff
@@ -0,0 +1,66 @@
+import { test, expect } from '@playwright/test'
+
+test('auths, fetches tracks, and favorites a track', async ({
+  page,
+  context
+}) => {
+  test.slow()
+
+  // Set entropy so we don't need to do OTP
+  const entropy = process.env.CREATE_AUDIUS_APP_TEST_ENTROPY ?? ''
+  await context.addInitScript((entropy) => {
+    if (window.location.hostname === 'audius.co') {
+      window.localStorage.setItem('hedgehog-entropy-key', entropy)
+    }
+  }, entropy)
+
+  await page.goto('localhost:4173')
+
+  const pagePromise = context.waitForEvent('page')
+  await page.getByRole('button', { name: 'Continue With Audius' }).click()
+
+  // Oauth popup
+  const authPage = await pagePromise
+  await authPage.waitForLoadState()
+  await authPage.getByRole('button', { name: 'Continue' }).click()
+
+  await expect(page.getByText('Continue With Audius')).not.toBeVisible()
+
+  // Fetch tracks
+  await page.getByRole('textbox').fill('createaudiusapptracks')
+  await page.getByRole('button', { name: 'Get Tracks' }).click()
```

Comment:
just curious, does this support `{name: /get tracks/i}`?

Reply:
@dylanjeffers It does support that syntax, yea

Reply:
Also the locators are case insensitive by default, need to do `exact: true` to make it case sensitive